templates/appliance/index.html

```diff
@@ -2,350 +2,169 @@
+                                alt="",
+                                width="852",
+                                height="480",
+                                hi_def=True,
+                                loading="auto",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+              <hr class="p-rule--muted" />
+              <a href="/appliance/openhab" class="p-heading--5">OpenHab</a>
+            </div>
+          </div>
+          <div class="col">
+            <div class="p-section--shallow">
+              <div class="p-image-container is-highlighted">
+                {{ image(url="https://assets.ubuntu.com/v1/9fd5be85-Plex.png",
+                                alt="",
+                                width="852",
+                                height="483",
+                                hi_def=True,
+                                loading="auto",
+                                attrs={"class": "p-image-container__image"}) | safe
+                }}
+              </div>
+              <hr class="p-rule--muted" />
+              <a href="/appliance/plex" class="p-heading--5">Plex</a>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
-    <div class="col-6">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/ebac33ef-hardware_easy.svg",
-        alt="",
-        height="85",
-        width="149",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-      <h2>
-        Confidently compatible
-      </h2>
-      <p class="p-heading--4">
-        Choose your certified board or box
-      </p>
-      <p>
-        It takes a team to make this work. Canonical and appliance publishers, hardware vendors and open source communities work together to ensure your appliance is updated, safely, for years to come.
-      </p>
-      <p>
-        We test all the certified combinations of appliance image and device, every time we change the system, for a decade or more. You can relax and let Ubuntu Core keep you up to date year after year with the latest features and fixes, and never miss a beat.
-      </p>
-      <p>
-        <a href="/certified/iot">See Canonical-certified hardware</a>
-      </p>
+  </section>
+
+  <section class="p-section--deep">
+    <hr class="is-fixed-width p-rule" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Turn your app
+          <br />
+          into an Ubuntu Appliance
+        </h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/c611f6cb-Image wrapper.png",
+                        alt="",
+                        width="1800",
+                        height="1013",
+                        hi_def=True,
+                        loading="auto",
```

Comment:
It doesn't look like this is in the vp on load, so we should probs lazy load instead
```suggestion
                        loading="lazy",
```